configs/webpack.common.config.js

```diff
@@ -39,6 +39,7 @@ module.exports = () => {
         hsd$: `${MODULES_DIR}/hsd/lib/hsd-browser`,
         bledger: `${MODULES_DIR}/bledger/lib/bledger-browser`,
         bmultisig: `${MODULES_DIR}/bmultisig/lib/bmultisig-browser`,
+        bcrytp: `${MODULES_DIR}/bcrypto`,
```

Comment:
typo

Reply:
I have no idea what you're talking about 🙄